src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -185,17 +196,49 @@ private void closeExecutorService() {
         }
     }
 
+    static MetricName metricName(Class<?> clazz, String name, LinkedHashMap<String, String> tags) {
+        String group = clazz.getPackageName();
+        String type = clazz.getSimpleName();
+        return metricName(name, type, group, tags);
+    }
+
     static MetricName metricName(Class<?> clazz, String name) {
         String group = clazz.getPackageName();
         String type = clazz.getSimpleName();
         return metricName(name, type, group);
     }
 
-    static MetricName metricName(String name, String type, String group) {
+    /**
+     * Generate an mBean metric name
```

Comment:
Is this comment correct, given it's actually returning a Yammer `MetricName`?

Reply:
Fair point, describing it as a Yammer `MetricName` is more accurate.

Reply:
Fixed in 242939b3d785e1a2833dd3aa2f942628861ad422